core/modules/user/user.module

```diff
@@ -849,7 +873,8 @@ function user_account_form(&$form, &$form_state) {
     $description = '';
     // Provide different help text if weak passwords are rejected.
     if ($reject_weak) {
-      $description = ' ' . t('Weak passwords will be rejected.');
+      //$description = ' ' . t('Weak passwords will be rejected.');
+      $description = ' ' . t('Password must be <em>@threshold</em> or stronger.', array('@threshold' => $threshold_label));
```

Comment:
Why are we prefixing this with a space?

Reply:
Maybe that's a holdover. Looks like it's not needed anymore.

Reply:
I think the space is a mistake in the previous if statement it makes sense but here `$description` is being overwritten so doesn't need a space at the beginning.

Reply:
You're right, that space is only needed `if (!$register)`, but here it's unnecessary. :+1: